spring-framework/5.3.14.wso2v1/src/main/resources/META-INF/spring.schemas

```diff
@@ -0,0 +1,66 @@
+# spring-beans
```

Comment:
Don't we need a license header here?

Reply:
Refered META-INF files of the previous spring orbit bundles and not available in there also. https://github.com/wso2/orbit/tree/master/spring-framework/3.2.9.wso2v2/src/main/resources/META-INF